dependencies.props

```diff
@@ -50,6 +50,12 @@
     <PlatformPackageVersion>2.1.0-preview1-25324-02</PlatformPackageVersion>
   </PropertyGroup>
 
+  <!-- Package versions used as toolsets -->
+  <PropertyGroup>
+    <FeedTasksPackage>Microsoft.DotNet.Build.Tasks.Feed</FeedTasksPackage>
+    <FeedTasksPackageVersion>1.0.0-prerelease-02121-01</FeedTasksPackageVersion>
```

Comment:
This doesn't match the dotnet/versions commit you're on:

https://github.com/dotnet/versions/blob/45d075f60f80600fb1a49944239c95bc839eeab4/build-info/dotnet/buildtools/master/Last_Build_Packages.txt

Microsoft.DotNet.Build.Tasks.Feed 2.1.0-prerelease-02417-06

I suggest running `/t:UpdateDependencies` to sync things up.

Reply:
Thanks, updated and amended.